federation/pkg/kubefed/join.go

```diff
@@ -49,7 +49,7 @@ const (
 
 var (
 	join_long = templates.LongDesc(`
-		Join a cluster to a federation.
+		Join adds a cluster to a federation.
```

Comment:
(No action required) Kind of begs the question of why the command pair isn't called 'Add/Remove'.